intro/http-server/Cargo.toml

```diff
@@ -18,7 +18,7 @@ opt-level = "z"
 [dependencies]
 anyhow       = "=1.0.75"
 embedded-svc = "=0.26.4"
```

Comment:
embedded-svc also needs updating in every place, thats why the CI fails to run this.